public/styles/_docs.scss

```diff
@@ -114,61 +114,36 @@
   }
 }
 
-.docs-tags {
```

Comment:
`.docs-tags`, `.docs-tag`, and `.docs h1 em` were unused relics. I simplified here by modifying the `.platform-label` class and extending it to be able to have different-coloured tags.